lib/pagseguro/subscription/response_serializer.rb

```diff
@@ -9,7 +9,52 @@ def initialize(xml)
 
       def serialize
         {}.tap do |data|
-          data[:code] = xml.at_css('directPreApproval > code').text
+          data[:code] = xml.css('directPreApproval > code').text
+        end
+      end
+
+      def serialize_from_search
+        {}.tap do |data|
+          data[:name] = xml.at_css('name').text
+          data[:code] = xml.css('code').text
+          data[:date] = xml.css('date').text
+          data[:tracker] = xml.css('tracker').text
+          data[:status] = xml.css('status').text
+          data[:reference] = xml.css('reference').text
+          data[:last_event_date] = xml.css('lastEventDate').text
+          data[:charge] = xml.css('charge').text
+          data[:sender] = serialize_sender if xml.at_css('sender')
```

Comment:
This line was a bit strange to understand at first reading, but it works :ok_hand: